ste/xfer-deleteBlobFS.go

```diff
@@ -51,7 +54,16 @@ func doDeleteHNSResource(jptm IJobPartTransferMgr) {
 	transferDone := func(err error) {
 		status := common.ETransferStatus.Success()
 		if err != nil {
-			status = common.ETransferStatus.Failed()
+			var respErr *azcore.ResponseError
```

Comment:
We need to add test cases for this and validate that 404 are ignored while others are reported.